### PR TITLE
New TestNet

### PR DESCRIPTION
**Pull request by vsuharnikov**
* New bridge address in EL: `0x0000000000000000000000000000000000006a7e`
* New EL genesis block with bridge for TestNet;
* Removed staking contract.

### CODE REVIEW

File: local-network/deploy/common-settings-show.ts [hunk 1]
```diff
@@ -1,2 +1,11 @@
 import * as all from "./src/common-settings";
+import * as n from "./src/nodes";
+
 console.log(all);
+
+const elBridgeAddressResponse = await n.wavesApi1.addresses.fetchDataKey(all.chainContract.address, 'elBridgeAddress');
+if (!elBridgeAddressResponse.value || elBridgeAddressResponse.type != 'string') throw new Error(`Unexpected value of "tokenId" contract key in response: ${elBridgeAddressResponse}`);
```
Comment (ivan-mashonskiy) [inline local-network/deploy/common-settings-show.ts:7]:
```suggestion
if (!elBridgeAddressResponse.value || elBridgeAddressResponse.type != 'string') throw new Error(`Unexpected value of "elBridgeAddress" contract key in response: ${elBridgeAddressResponse}`);
```

File: local-network/deploy/src/waves-txs.ts [hunk 2]
```diff
@@ -1,27 +1,6 @@
...
-        type: "string",
-        value: "%d%d%d%d__1__50000000__9__1000000"
-      }
-    ]
-  },
-  { privateKey: s.stakingContract.privateKey }
-)
-
 // cc - Chain contract
 // 2. Deploy chain contract script
```
Comment (ivan-mashonskiy) [inline local-network/deploy/src/waves-txs.ts:5]:
```suggestion
// 1. Deploy chain contract script
```

Comment (ivan-mashonskiy) [reply local-network/deploy/src/waves-txs.ts:5]: and also point 3 "Setup chain contract" below should be changed to point 2

File: src/test/resources/main.ride [hunk 3]
```diff
@@ -203,44 +176,43 @@ let (computedDelay, computedGenerator, computedTotalBalance, filteredMiners) = {
...
   }
-  FOLD<50>(allMiners, (INT_MAX, "", 0, []), processMiner)
+  FOLD<50>(allMiners, ("", 0, INT_MAX, []), processMiner)
 }
 
-func getChainLastBlockId(chainId: Int) = chainMeta(chainId)._2
-
-let (mclbIgnored1, mainChainEpoch, mainChainParentHash, mainChainGenerator) = blockMeta(mainChainLastBlock)
+let (mclbIgnored1, mainChainEpoch) = blockMeta(mainChainLastBlock)
+let (mainChainGenerator, mceIgnored1) = epochMeta(mainChainEpoch).valueOrErrorMessage("Can't find main chain epoch meta for epoch " + mainChainEpoch.toString())
```
Comment (ivan-mashonskiy) [inline src/test/resources/main.ride:187]: it looks like `mainChainGenerator` is not used, so we can delete it
```suggestion
```

File: local-network/deploy/common-settings-show.ts (hunk 1 above)
Comment (vsuharnikov) [reply local-network/deploy/common-settings-show.ts]: ✅

File: local-network/deploy/src/waves-txs.ts (hunk 2 above)
Comment (vsuharnikov) [reply local-network/deploy/src/waves-txs.ts]: ✅

File: src/test/resources/main.ride (hunk 3 above)
Comment (vsuharnikov) [reply src/test/resources/main.ride]: ✅

**Review by ivan-mashonskiy — APPROVED** [review]

**Review by phearnot — APPROVED** [review]

**Pull request merged**